foot-in-the-door/pages/video.js

```diff
@@ -0,0 +1,123 @@
+/** @format */
+
+import Head from 'next/head'
+import Layout from '../components/Layout'
+
+const videos = [
+  {
+    vimeoId: 480895859,
+    caption: 'Foot in the Door 5 Artist: Eyenga Bokamba',
+  },
+  {
+    vimeoId: 480525373,
+    caption: 'Foot in the Door 5 Artist: Sammy Austin',
+  },
+  {
+    vimeoId: 471541996,
+    caption: 'Foot in the Door 5 Artist Video: Ilene Krug Mojsilov',
+  },
+  {
+    vimeoId: 478029945,
+    caption: 'Foot in the Door 5 Artist: Davinia',
+  },
+  {
+    vimeoId: 475231372,
+    caption: 'Foot in the Door 5 Artist: Alonzo Pantoja',
+  },
+  {
+    vimeoId: 480470984,
+    caption: 'Foot in the Door 5 Artist: Tamara Aupaumut',
+  },
+  {
+    vimeoId: 478130564,
+    caption: 'Foot in the Door 5 Artist: TimmiLynn Johnson',
+  },
+  {
+    vimeoId: 479910635,
+    caption: 'Foot in the Door 5 Artist: Ken Wenzel',
+  },
+  {
+    vimeoId: 478085063,
+    caption: 'Foot in the Door 5 Artist: Paula Warn',
+  },
+]
+
+function video() {
+  return (
+    <Layout>
+      <main className="md:flex items-start mb-12">
+        <div className="md:w-1/2 mr-12">
+          <h1 className="text-3xl sm:text-4xl lg:text-5xl font-black tracking-wide">
+            Foot in the Door 5
+          </h1>
+          <h2 className="text-3xl sm:text-4xl lg:text-5xl font-light">
+            Artist Videos
+          </h2>
+          <p className="text-2xl font-light">
+            What makes "Foot in the Door 5: The Virtual Exhibition" so unique?
+          </p>
+          <p className="text-2xl font-light">
+            {' '}
+            The thousands of participating Minnesota artists! Get to know a few
+            of them in these 1-minute videos. Each artist touches on their
+            creative process and specific artwork featured in the exhibition.
+            Meet participating artists. New videos released each week.
+          </p>
+        </div>
+      </main>
+      <aside className="md:flex pb-6">
+        <div id="video" className="md:w-1/2 pb-8 md: pb-0 md:mr-12">
+          {videos.map((video) => (
```

Comment:
eslint again. In react, every item in a list needs to have a key: https://reactjs.org/docs/lists-and-keys.html

```
react/jsx-key: Missing "key" prop for element in iterator
```

Reply:
This was also updated. i'm not sure why it wasn't comitted. I'll update all of these.